bokeh/client/session.py

```diff
@@ -256,6 +261,7 @@ def close(self, why="closed"):
         self._connection.close(why)
 
     def loop_until_closed(self):
+        [cb.start() for cb in self._callbacks]
```

Comment:
That was my original implementation but I've changed for 2 main concerns: we start the callbacks before we are actually "done" creating the app and if you using a client session than your client session will be soon closed if you don't `loop_until_closed`..